tests/hla/process_randomlist.py

```diff
@@ -0,0 +1,204 @@
+import sys
+import traceback
+import os
+import datetime
+import pytest
+import numpy as np
+from astropy.table import Table, vstack
+from astropy.io import ascii
```

Comment:
Imported but not used...

Reply:
These are used - will keep.

Reply:
The packages 'ascii' and pytest' are not used anywhere in this code, so those imports can be removed.  The remaining import statements are used in the code and should obviously remain.